### ISSUE TITLE
Docs for tracking activities of the FR team

### ISSUE THREAD

**Issue by spapastamkou** [programminghistorian/jekyll#1247]
In #1136 it was suggested the FR team does a document [similar to this of the ES team](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/es/lista-de-traducciones.md) to keep its activities organized. 

As I went on with the editing process of the lessons currently under review, I created instead the following gsheets (with the same data more or less as in the ES doc, or the missing data can be added, for example lessons we do not wish to translate) to keep us organized. The reason was I found them flexible, easy and quicker to update: 
1) [Liste des traductions](https://docs.google.com/spreadsheets/d/1nj520xwTYnfxItzisGL096Oo28D-wOST82At2BZy8OI/edit#gid=0) (list of translations)
2) [Liste des contributeurs](https://docs.google.com/spreadsheets/d/1THqVCECRE1ZaGoLnTnX8Q_rSM0fvaWmugOHpLzSD9dI/edit#gid=0) (list of contributors). This second one is to have a quick global view of FR collaborators and their roles. Just in case we need it. 

Furthermore, I put all available info for [FR reviewers](https://docs.google.com/spreadsheets/d/1EMQyVpIYnWjNr4hEQDDilkUl28gGPA2gSLjp61_AFgM/edit#gid=682614077) and [FR translators](https://docs.google.com/spreadsheets/d/1EMQyVpIYnWjNr4hEQDDilkUl28gGPA2gSLjp61_AFgM/edit#gid=1025549187) in the PH Reviewer Tracking gsheet. 

Comments are welcome. Please let me know what you think and if further precisions are required, @fdlaramee and @mpuren, so we can agree on a common tool that fits all and can be sustainable. Once we are ok, I'll create a .md file with links and info in our submissions repository.

**Comment by fdlaramee** [comment]
@spapastamkou Looks perfect to me. We may add to "Liste des traductions" a section for untranslated lessons that are referenced in the ones that have already been translated, if we decide to (try to) give them higher priority in our process.

**Comment by spapastamkou** [comment]
@fdlaramee I made an addition in the list as a test, taken from your own lesson. If not ok, we can find a better way (after the weekend:))

**Comment by mpuren** [comment]
@spapastamkou I agree with @fdlaramee : we should have a for the lessons that are referenced in the translated lessons, and that could be considered as a priority. I have created [this list](https://docs.google.com/spreadsheets/d/1iVLbxx_LHqdoRQzuwFXIwaOVadhsOqJELOxF6sVDpoQ/edit?usp=sharing) (it is just a beginning).

**Comment by acrymble** [comment]
Are you documenting somewhere where these will be found, in case we forget in the future? Maybe a page on the Wiki? https://github.com/programminghistorian/jekyll/wiki

**Comment by mpuren** [comment]
@acrymble good idea, what do you think @fdlaramee and @spapastamkou ?

**Comment by spapastamkou** [comment]
@acrymble @mpuren The idea is to create a .md file in the [FR repository for translations](https://github.com/programminghistorian/ph-submissions/tree/gh-pages/fr/traductions), with the links to the gsheets and explanations, to have all in one place (as the ES team do with their lists). 

@mpuren The very last addition to the Liste des traductions [gsheet](https://docs.google.com/spreadsheets/d/1nj520xwTYnfxItzisGL096Oo28D-wOST82At2BZy8OI/edit#gid=0) is a test with a lesson title such as the ones you have in your list so that we can see if that fits. If you and @fdlaramee think we can state this type of priority translations (stated in the "traduction" column) like this, please add your links as well. Otherwise, open to ideas to find a better way (such as a separate list like the one you created.

**Comment by mpuren** [comment]
@spapastamkou  Thx for the clarification

I propose to create a new sheet in the liste des traductions [gsheet](https://docs.google.com/spreadsheets/d/1nj520xwTYnfxItzisGL096Oo28D-wOST82At2BZy8OI/edit?usp=sharing) : I have renamed the 1st sheet as "Traduits" and I have created  a 2nd sheet "A traduire prioritairement" where we can put the list of the lessons we should translate first (I will add those I've spotted). When one of these lessons has been translated, we delete its name from the sheet 2 after putting its name in the sheet 1.
In the 1st sheet, I've also added a column "Renvois vers des leçons non traduites (cf Feuille "A traduire prioritairement")" at the end of the table, where we can put the names of the lessons that are in the 2nd sheet. (I hope I'm clear!)

**Comment by spapastamkou** [comment]
I created a `liste-des-traductions.md` file in `ph-submissions` ([here](https://github.com/programminghistorian/ph-submissions/blob/gh-pages/fr/traductions/liste-traductions.md)). I also documented these lists so that anyone knows what to do.  

@mpuren: I just renamed the first sheet "Suivi des traductions" because it is not only for already translated tutorials (translated is only an attribute). For the 2nd sheet I added a column so that we know from which lesson comes the tutorial we attach priority to because mentionne in the first. 

For the rest, I propose to keep simple: instead of transfering data from one sheet to another (that we will probably forget at some point), we can just put a strikethrough (barrer) on the lessons of the second sheet if they are being translated. In any case, we will register them in the first sheet if in the process of translation/evaluation. The strikethrough will help keep track of the progress as well.  
In any case, if something does not fit us, we'll know at the process and we can look for a solution.
